docker-compose.dev.yml

```diff
@@ -66,7 +64,7 @@ services:
       - minio4
 
   postgres:
-      image: postgres
+      image: postgres:16
```

Comment:
I had a conflict with my existing postgres data
 ```
 postgres-1  | 2025-02-14 21:00:51.286 UTC [1] DETAIL:  The data directory was initialized by PostgreSQL version 14, which is not compatible with this version 16.7 (Debian 16.7-1.pgdg120+1).
postgres-1 exited with code 1
``` 
After deleting the postgres volume, it worked for me.